apps/api/src/teams/event-registrations/event-registrations.service.ts

```diff
@@ -71,28 +104,45 @@ export class TeamEventRegistrationsService {
     return await this.teamEventRegistrationRepository.findWithPagination(
       options,
       filter,
-      { orderBy: { createdAt: 'ASC' }, populate: ['user', 'event', 'event.team'] },
+      {
+        orderBy: { createdAt: 'ASC' },
+        populate: ['user', 'event', 'event.team'],
+      },
```

Comment:
Pourquoi l'avoir changé
```suggestion
      { orderBy: { createdAt: 'ASC' }, populate: ['user', 'event', 'event.team'] },
```